### ISSUE TITLE
Favorites apps and quit icons get smaller in Cinnamon menu when hovering over categories

### ISSUE THREAD

**Issue by P3p1t0** [adapta-project/adapta-gtk-theme#635]
**Reporter info**

 ```
 * Distribution - Manjaro Cinnamon 17.1.4 - Testing branch
 * Gtk+ 3.0 version - 3.22.26
 * Desktop environment - 3.6.7
 * Display server protocol - X11
 ```

**Adapta version (if you knew)**

Issue noticed on 3.93.0 (rev 106) and 3.93.0 (rev 113).

**Related Application and/or Shell theming (or name of widget)**

Cinnamon menu

**Actual issue**

When you hover over the categories in Cinnamon menu, the favorite apps and quit icons get smaller. Sometime, it turns back to the normal size very quickly and sometime, it will only turn back to normal when you hover over the entries of the selected category. When icons get smaller, categories and entries move to the left. When icons get back to normal, categories and entries return back to the right. It looks like the menu is shaking, which is not very pleasant to use.

You can read my report on the Manjaro forum for more details about that issue.

[https://forum.manjaro.org/t/testing-update-2018-02-08-kernels-browsers-plasma-gcc-nvidia-systemd/40022/29](https://forum.manjaro.org/t/testing-update-2018-02-08-kernels-browsers-plasma-gcc-nvidia-systemd/40022/29)
[https://forum.manjaro.org/t/testing-update-2018-02-08-kernels-browsers-plasma-gcc-nvidia-systemd/40022/35](https://forum.manjaro.org/t/testing-update-2018-02-08-kernels-browsers-plasma-gcc-nvidia-systemd/40022/35)
[https://forum.manjaro.org/t/testing-update-2018-02-08-kernels-browsers-plasma-gcc-nvidia-systemd/40022/42](https://forum.manjaro.org/t/testing-update-2018-02-08-kernels-browsers-plasma-gcc-nvidia-systemd/40022/42)

**Steps to reproduce (if you knew)**
A bit hardcore, but :

1. Install Manjaro 17.1.4 Cinnamon on a real computer (I have never succeed to reproduce this issue on VirtualBox);
2. Switch to testing branch with sudo pacman-mirrors -b testing.
3. Update the whole system with sudo pacman -Syyu;
4. Reboot.

I suppose just using Adapta theme for Cinnamon panel, then reboot, is more than enough.

**Expected behaviour**

Favorite apps and quit icon should always stay the same size; it shouldn't change size when hovering over something in particular.

**Other Note (feature-request, question, etc...)**

- Doesn't seem to be a packaging issue since I have this issue with adapta-maia-theme 3.93.0.113-1 (packaged by Manjaro Team, customized a bit to have turquoise color instead of blue) and with adapta-gtk-theme 3.93.0.106-1 (which is a package that directly comes from Arch Linux).
- Doesn't seem to be an icon issue since I still have the issue even when I use Adwaita icon theme instead of Vibrancy.
- Doesn't seems to be an Cinnamon issue since when I use another theme for Cinnamon Panel (Cinnamon or Vertex-maia), the "shaky" effect isn't present.

**Comment by tista500** [comment]
o_O?

Totally unreproducible on my locally build & installed current git tag https://github.com/linuxmint/Cinnamon/releases/tag/master.mint18 at least.

WTF...

**Comment by P3p1t0** [comment]
Although you weren't unable to reproduce the issue a few days ago, I still want to share with you the workaround I found for it on Manjaro.

- Installing and using Noto Sans Regular in Cinnamon (Menu > Preferences > Fonts) may fix the problem, but it isn't guaranteed; on one of my computers, it was not enough. 
- To fix the issue, I have a better success by ajusting the "Text scaling factor" settings from 1.0 to 1.1 in Cinnamon (Menu > Preferences > Fonts).

I didn't try to ajust font sizes without touching "Text scaling factor" settings at all.

[I reported my workaround on my distro's forum too.](https://forum.manjaro.org/t/testing-update-2018-02-08-kernels-browsers-plasma-gcc-nvidia-systemd/40022/52)

**Comment by tista500** [comment]
I've succeeded to reproduce this fuckin' weird behaviour now... :/

**Comment by P3p1t0** [comment]
Unfortunately, there's still something that isn't perfect.

I tested rev 128 with the package from my distro (Adapta Maia is slightly customized to have turquoise color). The "shaky" effect is mostly gone, but favorite apps and quit icons are smaller when I hover categories and are bigger when I hover favorite apps or when I hover entries of an category. However, as long as I stay in one of those zone, favorites apps and icons stay the same size; their size only changes when I move my cursor to a different zone.

The workaround I found still works.

Report on my distro's forum with screenshots : [https://forum.manjaro.org/t/testing-update-2018-02-15-kernels-firefox-dev-bluez-haskell-python/40450/25]( https://forum.manjaro.org/t/testing-update-2018-02-15-kernels-firefox-dev-bluez-haskell-python/40450/25)
